api-specificatie/zrc/openapi.yaml

```diff
@@ -72,83 +75,95 @@ paths:
+        zaakidentificatie:
+          title: Zaakidentificatie
+          description: >-
+            De unieke identificatie van de ZAAK binnen de organisatie die
+            verantwoordelijk is voor de behandeling van de ZAAK.
+          type: string
+          maxLength: 40
+        zaaktype:
+          title: Zaaktype
+          description: URL naar het zaaktype in de CATALOGUS waar deze voorkomt
+          type: string
+          format: uri
+          maxLength: 200
+          minLength: 1
         status:
-          description: URL naar de status van de zaak
+          title: Status
+          description: 'Indien geen status bekend is, dan is de waarde ''null'''
           type: string
+          format: uri
+          readOnly: true
+        omschrijving:
+          title: Omschrijving
+          description: Een korte omschrijving van de zaak.
+          type: string
+          maxLength: 80
+        # TODO: in het RGBZ 2.0 model loopt dit via de rollen/betrokkenen
+        # initiator:
+        #   title: Initiator
+        #   description: URL naar de initiator van de ZAAK.
+        #   type: string
+        #   format: url
         registratiedatum:
-          description: Tijdstip waarop de zaak is geregistreerd
-          type: string
-          format: 'date-time'
-        kanaal:
-          description: Kanaal via welke de zaak is gemeld
-          type: string
-        locatie:
-          description: Coordinate van de locatie waarop de zaak betrekking heeft in Well Known Text (WKT) formaat
+          title: Registratiedatum
+          description: De datum waarop de zaakbehandelende organisatie de ZAAK heeft geregistreerd
+          type: string
+          format: 'date'
+        # TODO: volgens RGBZ 2.0 komt kanaal uit KLANTCONTACT
+        # kanaal:
+        #   description: Kanaal via welke de zaak is gemeld
+        #   type: string
+        zaakgeometrie:  # NOTE: komt ook voor bij OBJECT
+          title: Zaakgeometrie
+          description: Punt, lijn of multi-vlak geometrie-informatie, in WKT formaat.
           type: string
         objecten:
+          title: OBJECTen
           description: Lijst met URL's naar aan de zaak gerelateerde objecten
           type: array
           items:
             type: string
-        zaakgegevens:
-          description: Domein specifieke gegevens beheorende bij de zaak
-          type: array
-          items:
-            $ref: '#/components/schemas/Zaakgegeven'
-      required:
-      - type
-      - omschrijving
-    ZaakId:
-      description: zaakidentificatie
-      properties:
-        zaakidentificatie: 
-          type: string
-          description: zaakidentificatie
-    Subject:
-      description: Een persoon of bedrijf
-      properties:
-        email:
-          description: e-mail adres
-          type: string
-          format: email
-        telefoonnummer:
-          description: telefoonnummer
-          type: string
-    Zaakgegeven:
```

Comment:
Mee eens dat het wat prematuur is om dit toe te voegen. Sowieso willen we zo min mogelijk van deze domein specifieke velden (wat volgens mij overeenkomt met extraElementen uit StUF-ZKN). Ik zelf had in Slack kort voorgesteld dat dit een  veld kon zijn met arbitraire JSON-data, om niet toe te werken naar een database-in-een-database. Voor je het weet gooit iedereen alles in dit veld...